package.json

```diff
@@ -4,6 +4,7 @@
   "repository": "github:internetarchive:openlibrary",
   "license": "AGPL-3.0",
   "scripts": {
+    "start": " webpack -w --mode=development",
```

Comment:
Rename to `watch`; `npm start` is usually used to start a server (not necessarily with watch); `watch` is more common for live compiling of JS/CSS resources.